samza-core/src/main/scala/org/apache/samza/container/TaskInstance.scala

```diff
@@ -218,7 +220,7 @@ class TaskInstance(
 
     trace("Checkpointing offsets for taskName: %s" format taskName)
```

Comment:
Doing that in the OffsetManager now. Since that message is at level DEBUG, it should appear whenever this one does. So prefer not to print all those offsets twice.